docs/ibmactionspz.md

```
@@ -0,0 +1,136 @@
+  (`tonistiigi/binfmt`) won't work. Not needed anyway since the arch is native.
+- **No `dmesg`** — `dmesg: read kernel buffer failed: Operation not permitted`.
+- **AppArmor restrictions on s390x** — podman's DNS resolution fails with
+  `socket: permission denied` because AppArmor blocks creating network sockets
+  in certain contexts
+  ([details](https://github.com/IBM/actionspz/issues/63#issuecomment-3662214924)).
+
+### No Homebrew
+
+Homebrew for Linux only supports x86_64 and ARM64:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)